src/CSFML/Network/TcpListener.cpp

```diff
@@ -102,3 +110,10 @@ sfSocketStatus sfTcpListener_accept(sfTcpListener* listener, sfTcpSocket** conne
 
     return status;
 }
+
+
+////////////////////////////////////////////////////////////
+unsigned short sfTcpListener_anyPort(void)
```

Comment:
```suggestion
unsigned short sfTcpListener_anyPort()
```
`(void)` is required in the headers because the headers are C but we can omit that in our C++ implementation files.